pyproject.toml

```diff
@@ -10,6 +10,16 @@ package-mode = false
 python = "^3.12"
 fastapi = "^0.110"
 uvicorn = "^0.29"
+debugpy = "^1.8.1"
```

Comment:
Do you need to be as specific as locking to particular patch versions for all these?

Reply:
first time using poetry and simply called poetry add for all the packages I needed. Will loosen all locks